phpstan.neon

```diff
@@ -1,6 +1,7 @@
 parameters:
-    checkMissingIterableValueType: false
-    checkGenericClassInNonGenericObjectType: false
+    ignoreErrors:
+        -
+            identifier: missingType.iterableValue
```

Comment:
Is this supposed to be on a separate line?

Reply:
Not especially. I've brought it onto the same line